actors/evm/src/interpreter/system.rs

```diff
@@ -311,6 +350,16 @@ impl<'r, RT: Runtime> System<'r, RT> {
             .context_code(ExitCode::USR_SERIALIZATION, "failed to decode state")?
             .context_code(ExitCode::USR_ILLEGAL_STATE, "state not in blockstore")?;
 
+        let transient_data_lifespan = state.transient_data.unwrap().transient_data_lifespan;
```

Comment:
This will panic if there are no transient slots.

Reply:
removing the options from transient_data_state and transient_data_lifespan helped, but also some extra checks needed to be added  https://github.com/filecoin-project/builtin-actors/commit/978b49d1e0f1e50fbecf020cadcfb1b5b8c4275a